src/app/shared/cookies/browser-klaro.service.ts

```diff
@@ -68,14 +80,22 @@ export class BrowserKlaroService extends KlaroService {
       this.klaroConfig.translations.en.consentNotice.description = 'cookies.consent.content-notice.description.no-privacy';
     }
 
+    const configurationToHide$: Observable<Pick<typeof klaroConfiguration, 'name'>[]> =
+      this.configService.findByPropertyName(this.GOOGLE_ANALYTICS_KEY)
+        .pipe(
+          getFirstCompletedRemoteData(),
+          map(remoteData => this.mapInvalidConfiguration(remoteData, this.GOOGLE_ANALYTICS_SERVICE_NAME)),
+          take(1)
+        );
+
     this.translateService.setDefaultLang(environment.defaultLanguage);
 
     const user$: Observable<EPerson> = this.getUser$();
 
     const translationServiceReady$ = this.translateService.get('loading.default').pipe(take(1));
 
-    observableCombineLatest([user$, translationServiceReady$])
-      .subscribe(([user, translation]: [EPerson, string]) => {
+    observableCombineLatest([user$, translationServiceReady$, configurationToHide$])
+      .subscribe(([user, translation, servicesToHide]: [EPerson, string, Pick<typeof klaroConfiguration, 'name'>[]]) => {
```

Comment:
I'd take the opportunity to rename `translation` to `_` here to highlight that we're not using its value and maybe reorder the array so it's at the back